cfn-resources/auditing/cmd/resource/resource.go

```diff
@@ -27,7 +28,7 @@ import (
 	log "github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
 	progress_events "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
```

Comment:
this package is already fixed so you can remove the package alias and just use progressevent

Reply:
updated